pyqtgraph/dockarea/Dock.py

```diff
@@ -198,7 +198,7 @@ def startDrag(self):
         self.drag.setMimeData(mime)
         self.widgetArea.setStyleSheet(self.dragStyle)
         self.update()
-        action = self.drag.exec_() if hasattr(self.drag, 'exec_') else self.drag.exec()
+        action = self.drag.exec() if hasattr(self.drag, 'exec') else self.drag.exec_()
```

Comment:
this just simplifies to `self.drag.exec` now

Reply:
Qt.py provides Qt5 shim for QApplication.exec() but not for QDialog.exec()

Reply:
I mean... the true and false cases are exactly the same in this diff

Reply:
nevermind, I misread it